repos/fdbt-site/src/design/modules/_components.scss

```diff
@@ -32,106 +27,115 @@
             transform: translateY(-50%);
             line-height: normal;
         }
-
         &__input {
             padding-left: 22px;
         }
     }
 }
-
 .flex-container {
     display: flex;
     align-items: flex-end;
 }
-
 .padding {
     padding-left: 15px;
 }
-
 .pagination {
     padding: 0;
 }
-
 .pagination__item {
     display: inline-block;
     list-style: none;
 }
-
 .pagination__link {
     display: block;
     padding: 5px 10px;
     text-decoration: none;
-
     &:hover,
     &:focus {
         background: #f8f8f8;
         outline: 3px solid #ffbf47;
     }
-
```

Comment:
Whitespace is still here?

Reply:
red highlight means it is removed

Reply:
I dont think we should see red or green highlights, ideally these spacings shouldnt change at all. But if its not an easy thing to revert, its ok.